scripts/ci/push-image.sh

```diff
@@ -22,6 +22,22 @@
                 RELEASE_TAG="$commit"
         fi
 
-        okteto build --platform "${PLATFORMS}" --build-arg VERSION_STRING="${RELEASE_TAG}" -t "okteto/okteto:${RELEASE_TAG}" -f Dockerfile .
+        
+        beta_prerel_regex="^beta\.[0-9]+"
+        prerel="$(semver get prerel "${RELEASE_TAG}")"
+
+        tags="okteto/okteto:${RELEASE_TAG},okteto/okteto:dev"
+        
+        # if release tag is  not empty, push the stable image
```

Comment:
Non-blocking comment: this if checks for the opposite condition as the above, so `RELEASE_TAG` will never be empty, so we could simplify and remove this if.